home.css

```diff
@@ -0,0 +1,640 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap");
+@import url("http://fonts.cdnfonts.com/css/cocogoose");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+  color: #272a31;
+}
+
+body {
+  background-color: #f5f5f5;
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-bar {
```

Comment:
The fixed nav looks cool 😎 But there is an overlaying issue here. The speakers are in front of nav. You can give z-index values to solve it 👍 
![Screen Recording 2023-02-07 at 7 40 55 PM](https://user-images.githubusercontent.com/25887253/217326804-0290fcfd-253b-4e00-851e-3d870bdcdc8b.gif)